### DISCUSSION TITLE
Document Vue3 Composition Api

### DISCUSSION THREAD

**Discussion by BombaraGianluca** [storybookjs/storybook#17608] [Ideas]
Good afternoon,

I would to ask you if storybook in the future can document composable file in Vue 3 ecosystem. 

*For example if have useSorting.js file. I think will be very cool if storybook can document method and reactive data inside the composition api file. As ArgsTable for props/event and slot. 

In this moment, for js file, i use jsdoc for generate documentation and jest for unit test. 
The result of both, i included in story in storybook. 
But i think the integrated solution will be better. 

You think this feacture* in the future can be integrated in storybook?
Thanks

**Comment by dlferro** [comment]
Reference to a great example of needing this type of support. https://stackoverflow.com/questions/70820238/how-to-reuse-composable-props-in-vue-3

**Comment by chakAs3** [comment]
Hi @BombaraGianluca  actually this is a very interesting idea, let me look into it in details, and see with the team for the eventual implementation   ........Thank you !
